templates/pages/category.html

```diff
@@ -24,16 +24,10 @@ <h1 class="page-heading">{{category.name}}</h1>
 {{{category.description}}}
 {{{snippet 'categories'}}}
 <div class="page">
-    {{#or category.subcategories category.faceted_search_enabled}}
+    {{#or category.subcategories category.faceted_search_enabled category.shop_by_price}}
```

Comment:
This should probably be a simple `if` check now right? Since it's not like an `or` / `else`.

Reply:
This has always been an `or` statement as I ended up reverting the changes from [1036](https://github.com/bigcommerce/cornerstone/pull/1036/files). I did try to change it to an `if` upon doing so though it did generate a 500 internal server error with stapler.

Reply:
Ah yeah that's my bad, I didn't understand what the `or` helper was actually doing.